ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -114,10 +121,12 @@ const AdminUser = withUrlParams()(class extends React.Component<Props, State> {
     this.state = {
       institutionsLoadingError: '',
       loading: true,
-      profile: null,
+      oldProfile: null,
```

Comment:
I am a little confused on the introduction of oldProfile. It seems to replace saveDisabled, but the saveDisabled state is still getting set in a few places, but never read. The oldProfile also does not get updated. A few questions:

1. When the profile is updated and saved, it seems like the save button will still be enabled (I may be wrong) - is this intentional? 
2. Would it be unreasonable to revert back to one profile state, and setting `saveDisabled` to `false` in `setVerifiedInstitutionOnProfile`?